store/mockstore/mocktikv/cop_handler_dag.go

```diff
@@ -43,6 +45,50 @@ import (
 
 var dummySlice = make([]byte, 0)
 
+// locCache is a simple map with lock. It stores all used timezone during the lifetime of tidb instance.
+// Talked with Golang team about whether they can have some forms of cache policy available for programmer,
+// they suggests that only programmers knows which one is best for their use case.
+// For detail, please refer to: https://github.com/golang/go/issues/26106
+type locCache struct {
```

Comment:
How about locationCache? locCache looks like lockCache.

Reply:
loc is generally used everywhere in our codebase. Additionally, it is generally used in `time` package.